types/rdf-validate-shacl/tsconfig.json

```diff
@@ -14,7 +15,12 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "paths": {
+            "@rdfjs/*": [
+                "rdfjs__*"
+            ]
+        }
```

Comment:
Why is this here, does it need to be applied to all packages?

Reply:
Without this DefinitelyTyped cannot resolve scoped packages. Compare with [any random types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/rebass__forms/tsconfig.json#L17-L20)